stack/keycloak/migration/04_client-service-account.yml

```diff
@@ -0,0 +1,9 @@
+id: client-service-account
+author: Default
+realm: local_realm
+changes:
+  - updateClient:
+      clientId: local
+      serviceAccountsEnabled: true
+
+# TODO test
```

Comment:
Why is a `TODO test` here? Is there something missing?

Reply:
Removed it.
Already tested it but forgot to remove it.